rest/replicator_test.go

```diff
@@ -5750,7 +5750,7 @@ func TestReplicatorDoNotSendDeltaWhenSrcIsTombstone(t *testing.T) {
 	base.RequireNumTestBuckets(t, 2)
 
 	defer db.SuspendSequenceBatching()()
-	base.SetUpTestLogging(t, base.LevelInfo, base.KeyAll)
+	base.SetUpTestLogging(t, base.LevelDebug, base.KeyHTTP, base.KeyHTTPResp, base.KeyChanges, base.KeySync, base.KeySyncMsg, base.KeyWebSocket, base.KeyWebSocketFrame, base.KeyDCP)
```

Comment:
Would it be useful to also have `KeyReplicate` here to see logs about conflicts and resolutions of them conflicts?

Reply:
At this point there's not much I'm filtering out, so I'll just replace with All

```suggestion
	base.SetUpTestLogging(t, base.LevelDebug, base.KeyAll)
```